website/docs/docs/api/persist.md

```diff
@@ -69,9 +71,26 @@ Below is the API of the `persist` helper function.
     Please see the [docs](#merge-strategies) below for a full insight and
     understanding of the various options and their respective implications.
 
+  - `migrations` (Object, _optional_)
+
+    Are used to transform persisted store state from one representation to
+    another. It contains the following properties:
+
+    - `migrationVersion` (number, _required_)
+
+      The migration version to execute up to, starting at 0.
+
+    - `[0..1..n]` (Function, _required_)
```

Comment:
@no-stack-dub-sack I'm having trouble figuring out how to express this prop name in the API docs... The description below explains whats going on, but any suggestions as to how to write it here?

Reply:
I get your point - reading this might be a little confusing.

Maybe we could illustrate the migration config - with supplying texts instead?

> - `migrations` (Object, _optional_)
> 
> This config is used to transform persisted store state from one representation to another. This object is keyed by version numbers, with migration functions attached to each version. A `migrationVersion` is also required for this object, to specify which version to target. 
> ```js
> persist(
>   {...},
>   {
>     migrations: {
>       migrationVersion: 2,
> 
>       1: (state) => { ... },
>       2: (state) => { ... },
>     },
>   }
> );
> ```

A little more context might make it easier to "get"?

Reply:
Updated!